src/components/calcite-pick-list/shared-list-logic.ts

```diff
@@ -84,7 +84,7 @@ export const sharedListMethods = {
   },
   deselectSiblingItems(this: CalcitePickList | CalciteValueList, item: pickOrValueListItem) {
     this.items.forEach((currentItem) => {
-      if (currentItem !== item) {
+      if (currentItem.value !== item.value) {
```

Comment:
This was trying to match a `CalcitePickListItem` to a `CalciteValueListItem`. Changed to use `value` since one wraps the other.

Reply:
Good catch.